lib/versioning/node/index.ts

```diff
@@ -1,22 +1,38 @@
 import { DateTime } from 'luxon';
 import { valid } from 'semver';
-import npm, { isValid, isVersion } from '../npm';
+import npm, { isVersion } from '../npm';
 import type { NewValueConfig, VersioningApi } from '../types';
-import { nodeSchedule } from './schedule';
+import { findScheduleForCodename, findScheduleForVersion } from './schedule';
 
 export const id = 'node';
 export const displayName = 'Node.js';
 export const urls = [];
 export const supportsRanges = false;
 
+function normalizeValue(value: string): string {
+  const schedule = findScheduleForCodename(value);
+  if (schedule) {
+    const major = schedule.version.replace('v', '');
+    return `^${major}.x`;
```

Comment:
```suggestion
    return `^${major}`;
```

Wouldn't either `^${major}` or `${major}.x` work?